libpod/container.go

```diff
@@ -886,6 +886,10 @@ func (c *Container) Export(path string) error {
 	c.lock.Lock()
 	defer c.lock.Unlock()
 
+	return c.export(path)
+}
+
+func (c *Container) export(path string) error {
 	if err := c.syncContainer(); err != nil {
```

Comment:
The syncContainer can go in Export(), I don't think we need it in export

Reply:
fixed